cookbook/docs/tutorial.rst

```diff
@@ -62,31 +68,37 @@ Steps
         res = say_hello(name=name)
         return res
 
-6. Update the simple test at the bottom of the file to pass in a name. E.g.
+The reason behind passing an argument is to help you get a hands-on experience as to how Flyte displays the inputs and outputs in its dashboard. 
+
+6. Update the simple test (function call) present at the bottom of the file to pass in a name. 
 
 .. tip::
 
   .. code-block:: python
 
-    print(f"Running my_wf(name='adam') {my_wf(name='adam')}")
+    print(f"Running my_wf(name='Flytekit') {my_wf(name='Flytekit')}")
```

Comment:
Should we say something different otherwise it will look the same as the hard coded string and not an easy test? Maybe name='Flyte python sdk'

Reply:
Done. Included the same.